sci-libs/primegen/primegen-0.0.97.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="Fast C/C++ prime number generator library"
+DESCRIPTION="primegen is a small, fast library to generate prime numbers \
+in order. It generates the 50847534 primes up to 1000000000 in just 8 \
+seconds on a Pentium II-350; it prints them in decimal in just 35 seconds. \
+primegen can generate primes up to 1000000000000000, although it is not \
+optimized for primes past 32 bits. It uses the Sieve of Atkin instead of the \
+traditional Sieve of Eratosthenes."
+HOMEPAGE="https://cr.yp.to/primegen.html"
+COPYRIGHT="1999, D. J. Bernstein"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="https://github.com/ohnx/haiku-primegen/archive/$portVersion.tar.gz"
+SOURCE_DIR="haiku-primegen-$portVersion"
+CHECKSUM_SHA256="9eb6945fbf9f632eaed680b134025e84cb52703c0da1549e1dc14b3a47e6b497"
+
+ARCHITECTURES="x86 x86_gcc2 x86_64"
+
+PROVIDES="
+    devel:primegen = $portVersion
+    cmd:primes = $portVersion
+    cmd:primegaps = $portVersion
+    "
+
+REQUIRES="
+        haiku
+        "
+
+BUILD_PREREQUIRES="
+        cmd:make
+        cmd:nroff
+        cmd:shar
+        cmd:gcc
+        "
+
+BUILD_REQUIRES="
+        haiku_devel
+        "
+
+BUILD()
+{
+    make
+}
+
+INSTALL()
+{
+    make setup
```

Comment:
This is what I used:
```
INSTALL()
{
	install -d $binDir $docDir $manDir/man1 $manDir/man3
	install -d $includeDir/primegen $developLibDir
	install -t $binDir primegaps primes primespeed
	install -t $docDir README
	install -t $manDir/man1 primegaps.1 primes.1 primespeed.1
	install -t $manDir/man3 primegen.3
	install -t $includeDir/primegen primegen.h uint32.h uint64.h
	install -T primegen.a $developLibDir/libprimegen.a

	# devel package
	packageEntries devel \
		$developDir \
		$manDir/man3
}
```

Reply:
Thanks for that!

The builtin `make setup` from the program compiles a number of extra programs that need the patching... I will try this instead, since those extra programs aren't necessarily needed.